kiwi-ng/cray-sles15sp3-barebones/values-cje.yaml.j2

```diff
@@ -80,27 +80,27 @@
 #vim: et sw=4 ts=4
 repos:
  -
-    details: "SUSE SLE15SP2 packages, DST mirror"
-    alias: "sle15-sp2-Module-Basesystem"
+    details: "SUSE SLE15SP3 packages, DST mirror"
+    alias: "sle15-sp3-Module-Basesystem"
     priority: "4"
     imageinclude: "true"
-    path: "{{ env.BLOBLET_OS }}/rpms/sles/15sp2-all/Products/SLE-Module-Basesystem/"
+    path: "{{ env.BLOBLET_OS }}/rpms/sles/15-SP3-all/Products/SLE-Module-Basesystem/"
```

Comment:
This is the only change where it wasn't a 2->3 change - just wanted to verify this dir name change is deliberate?

Reply:
It is. The location of the SP3 repo changed upstream (this is not for on system, this is for building in Jenkins).